uaclient/cli.py

```diff
@@ -596,6 +596,16 @@ def wrapper(*args, **kwargs):
                 logging.exception(exc.msg)
             print("{}".format(exc.msg), file=sys.stderr)
             sys.exit(exc.exit_code)
+        except Exception as exc:
+            with util.disable_log_to_console():
+                logging.exception(exc)
```

Comment:
As we've discussed before, we can pass in a message about the _context_ of the exception handling here, as we will get the details of the exception because we call `logging.exception`.

Reply:
Added the bookend log message for context since we already log the command itself we should be in good shape.

logging.exception("Unhandled exception, please file a bug")